### DISCUSSION TITLE
Handle multiple windows

### DISCUSSION THREAD

**Discussion by RobbeCl** [i3/i3#4594] [Q&A]
I started using I3 and I am trying to wrap my head around the whole I3-workstyle. At this moment I often work at 2 projects at once. They both have a frontend and backend (web application). Previously, I opened the frontend application on monitor 1 and the backend application on monitor 2. Then I did the same for project 2 but on a different workspace. The way, I could quickly navigate between them and also keep them separately. 

How would I do this the I3-way? Ideally I would have one level above a workspace, let's say a workspace group. But I don't know if I3 has this feature (I came across this: https://github.com/infokiller/i3-workspace-groups but I don't know if there is a 'native' way)?

Thanks!

**Comment by Airblader** [comment]
In i3, workspaces are separate per output, and there is no built-in way to synchronize switching across outputs. If your workflow requires that, it will require some external script or possibly at least chaining some i3 commands together to perform all workspace focus changes at once.
